QuantConnect.IBAutomater/IBAutomater.cs

```diff
@@ -204,9 +217,11 @@ public StartResult Start(bool waitForExit)
                         // initialization completed
                         else if (e.Data.Contains("Configuration settings updated"))
                         {
+                            // 2FA confirmation successful
+                            _twoFactorConfirmationPending = false;
```

Comment:
Not sure its worth setting it to `false`? It actually leaves a small window for a race condition if `_ibAutomaterInitializeEvent.WaitOne(TimeSpan.FromSeconds(60))` timedout

Reply:
Hmm, we'll still need to set it to `false` before the method returns, as it's called multiple times (once per restart).
Maybe at `L295` ?

Reply:
Oh right! sounds good, or also at the top of `Start`